crates/astria-bridge-withdrawer/local.env.example

```diff
@@ -31,10 +31,26 @@ ASTRIA_BRIDGE_WITHDRAWER_SEQUENCER_COMETBFT_ENDPOINT="http://127.0.0.1:26657"
 # Chain ID of the sequencer chain which transactions are submitted to.
 ASTRIA_BRIDGE_WITHDRAWER_SEQUENCER_CHAIN_ID="astria"
 
+# Set to true to enable frost threshold signing.
+ASTRIA_BRIDGE_WITHDRAWER_FROST_THRESHOLD_SIGNING_ENABLED=false
+
 # The path to the file storing the private key for the sequencer account used for signing
 # transactions. The file should contain a hex-encoded Ed25519 secret key.
+# Only used if `frost_threshold_signing_enabled` is false.
 ASTRIA_BRIDGE_WITHDRAWER_SEQUENCER_KEY_PATH=/path/to/priv_sequencer_key.json
 
+# The minimum number of frost participants required to sign a transaction.
+# Only used if `frost_threshold_signing_enabled` is true.
+ASTRIA_BRIDGE_WITHDRAWER_FROST_MIN_SIGNERS=0
+
+# The path to the json-encoded frost public key package.
+# Only used if `frost_threshold_signing_enabled` is true.
+ASTRIA_BRIDGE_WITHDRAWER_FROST_PUBLIC_KEY_PACKAGE_PATH=""
```

Comment:
Could we combine this with `ASTRIA_BRIDGE_WITHDRAWER_SEQUENCER_KEY_PATH`?

The naming is a bit off, but we could a) deprecate this config and b) provide an alias for the time being.

Reply:
don't think we should combine them, `ASTRIA_BRIDGE_WITHDRAWER_SEQUENCER_KEY_PATH` contains a private key while `ASTRIA_BRIDGE_WITHDRAWER_FROST_PUBLIC_KEY_PACKAGE_PATH` is the public key package (no secrets), so they're pretty different